Source/DafnyCore/Verifier/Translator.ExpressionTranslator.cs

```diff
@@ -1915,8 +1905,8 @@ public ExpressionTranslator(ExpressionTranslator etran, string modifiesFrame)
             case BinaryExpr.ResolvedOpcode.Union:
             case BinaryExpr.ResolvedOpcode.Intersection:
             case BinaryExpr.ResolvedOpcode.SetDifference:
-            case BinaryExpr.ResolvedOpcode.MultiSetUnion:
-            case BinaryExpr.ResolvedOpcode.MultiSetIntersection:
+              //case BinaryExpr.ResolvedOpcode.MultiSetUnion:
+              //case BinaryExpr.ResolvedOpcode.MultiSetIntersection:
```

Comment:
If these are no longer needed, delete the lines. But why are the multiset operations excluded and the set operations still included?

Reply:
The set operations are still included because there is a flag that can be turned on for sets (aggressive) which is used for frame conditions.
Without these, the tests compiling the Dafny runtime failed, see here
https://github.com/dafny-lang/dafny/actions/runs/4734312552/jobs/8403056164
I found that by reverting this code to its original form, the dafny runtime would verify again. Do you have an idea why?

Reply:
I was unable to reproduce the failure. However, I think it is correct to _omit_ the multiset operations here, because `RewriteInExpr` is supposed to be in synch with `TrInMultiSet_Aux`, which only rewrites `MultiSetDisplayExpr`.